lib/data/interceptor/auth_interceptor.dart

```diff
@@ -26,66 +25,32 @@ class AuthInterceptor extends InterceptorsWrapper {
   /// Called if token has expired.
   final VoidCallback onTokenExpired;
 
-  /// Whether request requires authentication or not.
-  bool isAuthenticatedPath(RequestOptions options) {
-    return !HttpClient.nonAuthenticatedPaths.contains(options.path);
-  }
-
```

Comment:
It was my understanding that in most cases we would use this in projects :)

Reply:
I have introduced another instance of dio where auth interceptor is not needed, because most of the time authoriztion server and resource server are different. So, We need two instances of dio

Reply:
> because most of the time authoriztion server and resource server are different.

Do you have an example of that? From my experience they are usually the same

Reply:
@pedromassango it's even available in your current project 😄

Reply:
@atiqmlbd yeah, but most of the projects I've worked on the auth server is the same 🙈  and the current project is actually the first one with a separate auth server. What I'm trying to say is, maybe it is not "most of the time" that this separated auth server is needed